docs/resources/etc_fstab.md.erb

```diff
@@ -0,0 +1,126 @@
+---
+title: About the etc_fstab Resource
+---
+
+# etc_fstab
+
+Use the `etc_fstab` InSpec audit resource to test information about all partitions and storage devices on a system.
+## Syntax
+
+An etc_fstab rule specifies a device name, its mount point, its mount type, the options its mounted with,
+its dump options, and the order the files system should be checked.
+
+## Syntax
+
+Use the where clause to match a property to one or more rules in the fstab file.
+
+    describe etc_fstab.where { device_name == 'value' } do
+      its ( 'mount_point' ) { should eq ['hostname'] }
+      its ( 'file_system_type' ) { should eq [['list']] }
```

Comment:
Can we change all of these to be `cmp` matchers as well? Shouldn't need the double-array treatment.